src/test/java/uk/ac/ebi/atlas/search/metadata/MetadataSearchDaoIT.java

```diff
@@ -0,0 +1,75 @@
+package uk.ac.ebi.atlas.search.metadata;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.jdbc.datasource.init.ResourceDatabasePopulator;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+import uk.ac.ebi.atlas.solr.cloud.SolrCloudCollectionProxyFactory;
+
+import uk.ac.ebi.atlas.solr.cloud.collections.SingleCellAnalyticsCollectionProxy;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@WebAppConfiguration
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = TestConfig.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class MetadataSearchDaoIT {
+    @Inject
+    private DataSource dataSource;
+
+    @Inject
+    private SolrCloudCollectionProxyFactory collectionProxyFactory;
+
+    private SingleCellAnalyticsCollectionProxy singleCellAnalyticsCollectionProxy;
```

Comment:
As this variable is just being used in `setUp` method so it would be better in my opinion to move it there as a local variable rather than a global variable.